### DISCUSSION TITLE
How do I update SideStore if no update appears?

### DISCUSSION THREAD

**Discussion by abovethecloud** [SideStore/SideStore#817] [Q&A]
I have recently installed SideStore 5.9. The problem is that 5.10 came out immediately after that, but the update is not detected by SideStore itself, so I’m unable to update SideStore from within itself. I’ve tried clearing the catch and rebooting the device to no avail.
What is the best course of action now to update SideStore? I’d rather not have to do it through the PC if at all possible.
Thanks in advance.

**Comment by ghost** [comment]
Just install the ipa using SideStore

**Reply by abovethecloud** [reply]
I remember I tried it a while ago in a previous version and it didn’t seem to work… Are you sure it’s a supported behaviour to install the SideStore IPA through SideStore itself? If so, I’ll try again
